internal/pkg/plugin/githubintegrations/githubintegrations.go

```diff
@@ -0,0 +1,255 @@
+package githubintegrations
```

Comment:
It's very similar to githubactions, can we reuse the code by some way.

Reply:
I agree with Daniel here. I think the name should at least have "trello" in it. Like "trellogithub" or something like that.

Reply:
will refactor later,  and will abstract out common code between githubaction and githubintegrations at that time

Reply:
Some thoughts: trellogh(github), trellogl(gitlab), etc.